common/jvm/src/test/scala/scalan/util/FileUtil.scala

```diff
@@ -6,10 +6,36 @@ import java.nio.file._
 import java.nio.file.attribute.BasicFileAttributes
 import scala.Console
 import scala.collection.JavaConverters._
-import scalan.util.StringUtil.StringUtilExtensions
+import scalan.util.StringUtil.{StringUtilExtensions, cleanFileName}
```

Comment:
StringUtilExtensions import not used

import scala.Console    as well

Reply:
fixed (except Console, which is used)